docs/user-guide/api-mediation/configuration-personal-access-token.md

```diff
@@ -1,13 +1,17 @@
-# Enabling single sign on for clients via personal access token configuration 
+# Enabling single sign on for clients via Personal Access Token configuration 
 
 
 :::info Roles: system programmer, system administrator, security administrator
 :::
 
-Use the following procedure to enable personal access tokens.
+Use the following procedure to enable Personal Access Tokens.
 
 1. Open the file `zowe.yaml`.
 2. Find or add the property with the value `components.gateway.apiml.security.personalAccessToken.enabled: true`.
 3. Restart Zowe.
 
-For more information about using personal access tokens, see [Authenticating with a Personal Access Token](./authenticating-with-personal-access-token.md).
\ No newline at end of file
+For more information about using Personal Access Tokens, see [Authenticating with a Personal Access Token](./authenticating-with-personal-access-token.md).
+
+:::note
+In order to enable the Personal Access Tokens support when using the Caching Service, **Infinispan** is required as the storage solution.
+:::
```

Comment:
I would try to reinforce the fact that infinispan is part of the Zowe installation and no additional software or instalaltion is required, which is something that is sometimes misunderstood about it